GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Kroxylicious Governance
+
+This document outlines the governance model for Kroxylicious.
+This governance model is designed to uphold the principles of transparency, open collaboration, and community involvement.
+
+[cfc]: https://www.commonhaus.org/bylaws/cf-council.html
+[cdm]: https://www.commonhaus.org/bylaws/decision-making.html
+[ctp]: https://www.commonhaus.org/policies/trademark-policy/
+[coc]: ./CODE_OF_CONDUCT.md
+[contrib]: ./CONTRIBUTING.md
+
+## Table of Contents
+
+- [Roles and Responsibilities](#roles-and-responsibilities)
+- [Decision Making](#decision-making)
+- [Code of Conduct](#code-of-conduct)
+- [Trademark Policy](#trademark-policy)
+- [Contributing](#contributing)
+
+## Roles and Responsibilities
+
+- **Code owners:** Individuals with write access to the repository, responsible for reviewing and merging contributions, and steering project direction. These individuals (or groups) will be maintained in each repository (`.github/CODEOWNERS`).
```

Comment:
```suggestion
- **Code owners:** Individuals with write access to any repository within the GitHub organisation, responsible for reviewing and merging contributions, and steering project direction. These individuals (or groups) will be maintained in each repository (`.github/CODEOWNERS`).
```

> within the GitHub organisation

Not entirely sure we should reference GitHub here, but equally its the only place where the organisation is really manifest.